api/src/main/java/org/apache/cloudstack/api/command/user/project/CreateProjectCmd.java

```diff
@@ -116,7 +116,7 @@ public long getEntityOwnerId() {
             throw new InvalidParameterValueException("Account name and domain id must be specified together");
         }
 
-        if (userId != null && (accountId == null && domainId == null)) {
+        if (userId != null && (accountId == null || domainId == null)) {
```

Comment:
why should they both be specified? user + either implies the other

Reply:
ignore, bad reading on my part

Reply:
@DaanHoogland this was the requirement in the FR doc

Reply:
discussed off-line with @Pearl1594 . no functional need but bugs may arise when not implemented this way